projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -453,6 +453,48 @@ public static function is_supported_jetpack_recurring_payments() {
 		);
 	}
 
+	/**
+	 * Whether site has any paid plan.
+	 *
+	 * @param string $type - Type of a plan for which site is configured. For now supports empty and newsletter.
+	 */
+	public static function has_configured_plans_jetpack_recurring_payments( $type = '' ) {
```

Comment:
This function could be combined with the subscequent one in some way - they are duplicating some code

Reply:
Not sure how best to combine these two functions as the queries are different